### DISCUSSION TITLE
Strongswan upgrade not working

### DISCUSSION THREAD

**Discussion by IndhumathiAyyasamy** [strongswan/strongswan#1773] [Q&A]
Hi All,

In the linux version 18.04 (UBUNTU), tried installing strongswan beta version 6.0.0, the installation was successful.
But, the ipsec binaries were pointing to older version Strongswan U5.6.2.
Have seen the below linking need to be done during installation, have done that too (using export command), but that doesnot help here.
Can someone point if there is something, that need to be done explicitly to link to the latest strongswan.

<img width="469" alt="Strongswan_upgrade" src="https://github.com/strongswan/strongswan/assets/138560587/696b9dff-50eb-4c7d-b3e6-87345a2249b0">

**Comment by tobiasbrunner** [comment]
It's obviously not a good idea to install from sources while you still have distribution packages installed. So please completely remove all strongSwan related Ubuntu packages and possibly re-install from the source dir.

**Reply by IndhumathiAyyasamy** [reply]
Thanks for the reply.
No explicit version of Strongswan was installed prior to this beta version.
Linux is imaged and on top of that directly we are installing Strongswan beta 6.0 version only.
Not sure how it is pointing to U5.6.2
Could you please elaborate on what needs to be done here.

**Reply by tobiasbrunner** [reply]
> No explicit version of Strongswan was installed prior to this beta version.
> Linux is imaged

Then your image, wherever you got it, probably has strongSwan installed. Again, uninstall these packages with `apt-get purge <packages>` (have a look at the [source package for Ubuntu 20.04](https://packages.ubuntu.com/source/focal/strongswan) for a list of possible packages, note that 18.04 is EOL).
